tests/test_result.py

```diff
@@ -201,10 +201,26 @@ def test_expect_raising(self, exc_cls: t.Type[Exception]) -> None:
 
         assert msg in str(exc_info.value)
 
+    @pytest.mark.parametrize("exc_cls", (None, IOError))
+    def test_raise_if_err_raising(self, exc_cls: t.Type[Exception]) -> None:
+        """Test raise_if_err for Err() values."""
+        exp_exc = exc_cls if exc_cls else RuntimeError
+        kwargs = {"exc_cls": exc_cls} if exc_cls else {}
+        msg = "not what I expected"
+
+        with pytest.raises(exp_exc) as exc_info:
+            Err(2).raise_if_err(msg, **kwargs)
+
+        assert msg in str(exc_info.value)
```

Comment:
When I first wrote this and the above test, I wasn't including the stringified error value in the string used to instantiate the error class, so it's not tested for here. I think we should include it in both tests, e.g.: 

```suggestion
        assert msg in str(exc_info.value)
        assert "2" in str(exc_info.value)
```

Reply:
👍